### ISSUE TITLE
prefer second position of output_file_grp for AlternativeImage, make FILEGRP_IMG fallback

### ISSUE THREAD

**Issue by bertsky** [cisocrgroup/cis-ocrd-py#12]
Fixed by OCR-D/ocrd_tesserocr#59 for binarization already, this needs to be done for all processors. Moreover, this must be reflected in the documentation (ocrd-tool.json, README, docstrings) and tests.

Please assign @kba for this (after #11 is merged, so he can start a new PR).
